acquia/blt
# 9.2.x: Upgrade BLT to Xenial on Travis CI

Changes proposed:
---------
(What are you proposing we change?)

- Ubuntu 14.04 (Trusty Tahr) reaches EOL in April 2019 

- Upgrade Travis CI to use Ubuntu 16.04 (Xenial Xerus) 

- First commit is PR https://github.com/acquia/blt/pull/3333, so I could test on Travis CI before submitting this PR

- Xenial on Travis CI needs MySQL to be started with `services: mysql`
   * https://blog.travis-ci.com/2018-11-08-xenial-release

- Xenial on Travis CI needs Xvfb to be started with `services: xvfb` and not via `/etc/init.d/xvfb start`
   * https://blog.travis-ci.com/2018-11-08-xenial-release



Steps to replicate the issue:
----------
(If this PR is not fixing a defect/bug, you can remove this section.)

1. Use BLT in a Travis build using `dist: xenial`
Error: 
```
# Initialize xvfb (see https://docs.travis-ci.com/user/gui-and-headless-browsers)
# sudo apt-get install xvfb
export DISPLAY=:99.0
sh -e /etc/init.d/xvfb start
sh: 0
```

Steps to verify the solution:
-----------
(How does someone verify that this does what you think does?)

1. Update BLT to support Xenial on Travis CI
2. See this PR for pass. Also the intermediate commits on my fork which failed:
   * https://travis-ci.org/hugovk/blt/builds/479812999 fail
   * https://travis-ci.org/hugovk/blt/builds/479817942 fail
   * https://travis-ci.org/hugovk/blt/builds/479820941 fail
   * https://travis-ci.org/hugovk/blt/builds/479849834 pass
